packages/pages/src/common/src/template/internal/getTemplatesFilepath.test.ts

```diff
@@ -0,0 +1,42 @@
+import path from "path";
+import { Path } from "../../project/structure.js";
+import { getTemplatesFilepath } from "./getTemplatesFilepath.js";
+
+const domainPath = "src/templates/some.domain.com";
+const rootPath = "src/templates";
+jest.mock("glob", () => {
+  return {
+    sync: (path: string) => {
+      return path.indexOf(domainPath) != -1
+        ? [`${domainPath}/brand.tsx`, `${domainPath}/test.tsx`]
```

Comment:
Can you add a test with a second brand

Reply:
I updated the mocked glob to include additional filepaths from another domain folder and use `minimatch` as a dev util package to convert the glob expression to regex to simulate the filepath matching process (to avoid having to manually construct the folder structure and files for the test).
This should help test that it shouldn't retrieve files from domain 2 if we specify domain 1. As such, there's no need for adding another test as it will be the same as the second test we have here.